webhook/src/middleware/auth.middleware.js

```
@@ -20,8 +23,11 @@ async function authMiddleware(req, res, next) {
   try {
     const decoded = await getAuth().verifyIdToken(token);
     req.user = {
-      uid:   decoded.uid,
+      uid: decoded.uid,
       email: decoded.email,
+      name: decoded.name,
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)